profile_library/eq-3/HmIP-FSM/model.json

```diff
@@ -1,18 +1,21 @@
 {
-  "measure_description": "Manually measured",
-  "measure_method": "manual",
-  "measure_device": "From manufacturer specifications",
+  "measure_description": "Manually measured with dummy load attached at 7.7W (average over 300s) multiple times, than calculated average of that.",
+  "measure_device": "Shelly PM Mini Gen3 (S3PM-001PCEU16)",
+  "measure_method": "script",
+  "measure_settings": {
+    "VERSION": "v1.17.1:docker"
+  },
   "name": "HmIP-FSM",
   "standby_power": 0.2,
-  "standby_power_on": 0.2,
+  "standby_power_on": 0.74,
   "sensor_config": {
     "power_sensor_naming": "{} Device Power",
     "energy_sensor_naming": "{} Device Energy"
   },
   "device_type": "smart_switch",
   "calculation_strategy": "fixed",
   "only_self_usage": true,
-  "created_at": "2024-09-06T18:37:40",
+  "created_at": "2025-002-04T18:37:40",
```

Comment:
you don't have to change created_at per se. And there is a format issue, `002`

Reply:
ok. reverted this line